src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/partialPayment.js

```diff
@@ -36,6 +36,12 @@ function makePartialPayment(req, res, next) {
       response.order.remainingAmount.currency,
     ).divide(100);
     response.remainingAmountFormatted = remainingAmount.toFormattedString();
+
+    const discountAmount = new Money(
+      response.amount.value,
+      response.amount.currency,
+    ).divide(100);
```

Comment:
does this calculation check out for any currency? not just the ones using cents?

Reply:
do we already have a way to handle all currencies? or do we need to create something new
I tested with USD, Euro, and Indian rubees and they work fine
Japenese Yen however doesn't need this division

Reply:
You could have a look at the `getCurrencyValueForApi` method in AdyenHelper